### PR TITLE
FR-14232 - Step up onRedirectTo support

### PR DESCRIPTION
**Pull request by doregg**
When the user app includes a fallback route, we need to avoid using the react-router, because it shows the fallback instead of frontegg step up route (because we are navigating from an internal app page which is not frontegg route)

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "build:watch": "rm -rf dist && mkdir dist && rollup -w -c ./rollup.config.js"
   },
   "dependencies": {
-    "@frontegg/js": "6.159.0",
-    "@frontegg/react-hooks": "6.159.0"
+    "@frontegg/js": "6.155.0-alpha.8",
```
Comment (doregg) [inline packages/react/package.json:13]: revert it!

File: packages/react/package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
+    "@frontegg/react-hooks": "6.155.0-alpha.8"
```
Comment (doregg) [inline packages/react/package.json:14]: revert it!

File: yarn.lock [hunk 3]
```diff
@@ -2378,54 +2378,54 @@
   resolved "https://registry.yarnpkg.com/@frontegg/entitlements-javascript-commons/-/entitlements-javascript-commons-1.0.1.tgz#038a5b86166bd00cfd3afa942de361434c34ea04"
   integrity sha512-Ny9M9wLcA6/p9OD09plK3ETy5zyTWpgFoIorAYYYxk4J3S6ardsUOai2MlBmXqbG9J1uxD66rdFF5qr4bwqgyw==
 
-"@frontegg/js@6.159.0":
-  version "6.159.0"
-  resolved "https://registry.yarnpkg.com/@frontegg/js/-/js-6.159.0.tgz#3e8b4c8205d0b9a398173b6119adeb721abfaa9b"
-  integrity sha512-EW1Thw64AbaBzUfmKs9+0QZjOixRZKRLjNUP3/2422r5TSd+ED1X4x5hsqbg5lUCLS8OJJ0u+MuNUKMJfnrEjg==
+"@frontegg/js@6.155.0-alpha.8":
```
Comment (doregg) [inline yarn.lock:2381]: revert it!

File: packages/react/src/FronteggProvider.tsx [hunk 4]
```diff
@@ -47,6 +47,14 @@ export const Connector: FC<ConnectorProps> = ({ history, appName, isExternalHist
     if (opts?.preserveQueryParams || isAuthRouteRef.current(path)) {
       path = `${path}${window.location.search}`;
     }
+
+    const stepUpUrl = props.authOptions?.routes?.stepUpUrl;
```
Comment (yuvalotem1) [inline packages/react/src/FronteggProvider.tsx:51]: add options `byPassFrameworkRouter` and use it from the SDK
Another option is to create list of `routesToByPassReactRouter` and then
```
if(routesToByPassReactRouter.some(route=> path.startsWith(route))){
      window?.history?.pushState(null, '', path);
      return;
}
```

**Review by yuvalotem1 — COMMENTED** [review]
Why this issue is not happening for us with login routes in our portal? 
What is different in the portal settings?

File: packages/react/src/FronteggProvider.tsx (hunk 4 above)
Comment (doregg) [reply packages/react/src/FronteggProvider.tsx]: fixed

File: yarn.lock [hunk 5]
```diff
@@ -2415,9 +2415,6 @@
   version "3.1.48"
   resolved "https://registry.yarnpkg.com/@frontegg/rest-api/-/rest-api-3.1.48.tgz#78947faa40dca6a7aac27928e7ef3a62a8f29159"
   integrity sha512-/fcBufYuGL1zOeC9q2dGvXaNPxwff4Ppol7aHZDR3cYSTli6iaOAk+h1qNBTYWR3s20oRGMRR4UR8epROGyU/g==
-  dependencies:
```
Comment (doregg) [inline yarn.lock:2418]: Check why

**Pull request merged**